x-pack/test_serverless/api_integration/test_suites/observability/config.feature_flags.ts

```diff
@@ -26,5 +26,5 @@ export default createTestConfig({
 
   // include settings from project controller
   // https://github.com/elastic/project-controller/blob/main/internal/project/observability/config/elasticsearch.yml
-  esServerArgs: ['xpack.ml.dfa.enabled=false', 'xpack.ml.nlp.enabled=false'],
+  esServerArgs: ['xpack.ml.dfa.enabled=false', 'xpack.ml.nlp.enabled=true'],
```

Comment:
Only the security configs should be adjusted - please revert the changes for all the observability configs.

Reply:
Ahh, overly eager search and replace.  On it.

Reply:
Ahh, overly eager search and replace.  On it.

Reply:
[Here it is](https://github.com/elastic/kibana/pull/180458/commits/6a43d4636c9ccfa96636fc6f9ce41bda65d8bc22) boss